module/VuFind/src/VuFind/AjaxHandler/NotificationsReorder.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * AJAX handler to change the order of notifications.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) effective WEBWORK GmbH 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Controller
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Johannes Schultze <schultze@effective-webwork.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+namespace VuFind\AjaxHandler;
+
+use Laminas\Mvc\Controller\Plugin\Params;
+use VuFind\Db\Table\NotificationsBroadcasts;
+use VuFind\Db\Table\NotificationsPages;
+
+/**
+ * AJAX handler to change the order of notifications.
+ *
+ * @category VuFind
+ * @package  AJAX
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Johannes Schultze <schultze@effective-webwork.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class NotificationsReorder extends AbstractBase
+{
+    /**
+     * Database table for pages
+     *
+     * @var NotificationsPages
+     */
+    private $pagesTable;
```

Comment:
As a general rule, our codebase avoids `private` properties and uses `protected` everywhere to more easily facilitate extending/overriding code. Might be worth replacing the private with protected everywhere, unless you feel there's a strong reason not to.